README.md

```diff
@@ -4,7 +4,7 @@
 
 <h1 align="center">The .NET Compiler Platform</h1>
 
-<p align="center"><a href="https://gitter.im/dotnet/roslyn?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge&amp;utm_content=badge" rel="nofollow"><img            src="https://camo.githubusercontent.com/5dbac0213da25c445bd11f168587c11a200ba153ef3014e8408e462e410169b3/68747470733a2f2f6261646765732e6769747465722e696d2f4a6f696e253230436861742e737667" alt="Join the chat at https://gitter.im/dotnet/roslyn" data-canonical-src="https://badges.gitter.im/Join%20Chat.svg" style="max-width:100%;"></a> <a href="http://aka.ms/discord-csharp-roslyn" rel="nofollow"><img src="https://camo.githubusercontent.com/1ea6a95121cbf4179d411e853681838825392a7f0ae7e6bb1e03f4ea37c8fd5d/68747470733a2f2f646973636f72646170702e636f6d2f6170692f6775696c64732f3134333836373833393238323032303335322f7769646765742e706e67" alt="Chat on Discord" data-canonical-src="https://discordapp.com/api/guilds/143867839282020352/widget.png" style="max-width:100%;"></a></p>
+<p align="center"><a href="https://gitter.im/dotnet/roslyn?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge&amp;utm_content=badge" rel="nofollow">![Join the chat at https://gitter.im/dotnet/roslyn](docs/img/gitter.png)</a> <a href="http://aka.ms/discord-csharp-roslyn" rel="nofollow">![Chat on Discord](docs/img/discord-mark-white.png)</a></p>
```

Comment:
I _thought_ this would work, but I'm guessing either 

* my relative URL is wrong
* the relative URL isn't checking the source branch when I do rich diff (I doubt it though...)


Does someone know how to do this? Otherwise I can do similar as before and just have github usercontent links. I figured uploading the resources would be slightly better but am unsure

Reply:
The problem is you are mixing HTML and Markdown. Cannot you simply do this?
```suggestion
<p align="center"><a href="https://gitter.im/dotnet/roslyn?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge&amp;utm_content=badge" rel="nofollow"><img title="Join the chat at https://gitter.im/dotnet/roslyn" src="docs/img/gitter.png" /></a> <a href="http://aka.ms/discord-csharp-roslyn" rel="nofollow"><img title="Chat on Discord" src="docs/img/discord-mark-white.png" /></a></p>
```

Reply:
oh duh, for some reason I thought you could mix them. Thanks!

Reply:
You can mix them, but with caution. Adding newlines to separate the HTML and Markdown would make the markdown inside HTML be recognized properly, but newlines also introduce paragraphs which would break the layout here...